tracing/src/lib.rs

```diff
@@ -174,6 +174,24 @@
 //! # fn main() {}
 //! ```
 //!
+//! For functions which don't have built-in tracing support and can't have
+//! the `#[instrument]` attribute applied (such as from an external crate,
+//! the [`Span` struct][`Span`] has a [`in_scope()` method][`in_scope`]
+//! which can be used to easily wrap synchonous code in a span.
+//!
+//! For example:
+//! ```rust
+//! # #[cfg(not(feature = "no-std"))]
+//! # fn doc() -> Result<(), ()> {
+//! # mod serde_json {
+//! #    pub(crate) fn from_slice(buf: &[u8]) -> Result<(), ()> { Ok(()) }
+//! # }
+//! # let buf = &[];
+//! let json = info_span!("json.parse").in_scope(|| serde_json::from_slice(&buf))?;
```

Comment:
I think the issue here is not actually related to the `std` feature flag. Instead, the problem is that the `info_span!` macro is not imported.

We can fix this example either by changing it to 
```suggestion
//! let json = tracing::info_span!("json.parse").in_scope(|| serde_json::from_slice(&buf))?;
```
or by adding
```rust
use tracing::info_span;
```
at the top of the doctest.